astropy/units/core.py

```diff
@@ -2596,8 +2556,7 @@ def def_unit(
     exclude_prefixes=[],
     namespace=None,
 ):
-    """
-    Factory function for defining new units.
+    """Define a new unit.
```

Comment:
Maybe "Define a new unit, as well as possible prefix versions of it." 

Or maybe better add a second line:
```
Similar to defining a unit via `Unit` or `IrreducibleUnit`, but allowing
also to automatically generate prefix units in the given namespace.
```